std_msgs/Header header

int32 UNAVAILABLE = 1
int32 DISABLED = 2
int32 ACTIVE = 3

int32 state
